source/guide_codimd.rst

```diff
@@ -28,182 +30,266 @@ Inspired by Hackpad, with more focus on speed and flexibility, and built from Ha
   * :manual:`domains <web-domains>`
   * :manual:`Node <lang-nodejs>` and its package manager :manual_anchor:`npm <lang-nodejs.html#npm>`
 
-
 Prerequisites
 =============
 
-.. include:: includes/my-print-defaults.rst
+Node version
+------------
 
 We're using :manual:`Node <lang-nodejs>` in the stable version 8:
 
-::
+.. code-block:: console
 
- [isabell@stardust ~]$ uberspace tools version show node
- Using 'Node.js' version: '8'
- [isabell@stardust ~]$
+  [isabell@stardust ~]$ uberspace tools version show node
+  Using 'Node.js' version: '8'
+  [isabell@stardust ~]$
+
+MySQL credentials
+-----------------
+
+.. include:: includes/my-print-defaults.rst
 
-Your blog URL needs to be setup:
+URL setup
+---------
+
+Your URL needs to be set up:
 
 .. include:: includes/web-domain-list.rst
 
-Install the package manager ``yarn`` using the node.js package manager (``npm``):
+See :manual:`domains <web-domains>` for setting up a custom URL like ``write.example.com``:
```

Comment:
All guides should use `isabella.uber.space` as an example domain, that does not need to be added manually.

Reply:
I know, that is why I included the instructions for `isabell.uber.space` as required. In addition I want users to be able to configure the service for a custom domain.

Reply:
Do the steps for `isabell.uber.space` differ from `write.example.com`, except for the domain name? Adding domains is explained [in the manual](https://manual.uberspace.de/web-domains.html) and shouldn't be repeated in guides. Is there a reason for a custom domain I seem to be missing?

Reply:
Yes, they differ in two cases and including this additional information makes the setup easier:

**(1)** While `isabell.uber.space` is set up automatically when creating a Uberspace account, a custom domain needs to be set up manually. Of course, this information can be looked up in the Uberspace manual, but since this section of the guide is called "URL setup", I want to provide the user with the necessary command for setting up their URL - whatever it might be.

**(2)** In the guide's section called "[Configure web backend](https://deploy-preview-451--uberlab.netlify.com/guide_codimd.html#configure-web-backend)", at least for me, it was not obvious how to configure the web backend of a custom domain. The same argument as for (1) applies here as well.

I am not familiar with the other guides you host as part of UberLab 7. If adding this additional information goes against the style you use, I will remove the information. Yet, I would much rather keep it in there and make the guide easier to follow.

Reply:
Please have a look at commit 15502cc. It should resolve this issue.